submodules/misc/misc.js

```diff
@@ -402,6 +402,58 @@ define(function(require) {
 						}
 					}
 				},
+				'set_alert_info[]': {
+					name: self.i18n.active().oldCallflows.set_ringtone,
+					icon: 'play',
+					category: self.i18n.active().oldCallflows.advanced_cat,
+					module: 'set_alert_info',
+					tip: self.i18n.active().oldCallflows.set_ringtone_tip,
+					data: {
+						alert_info: ''
+					},
+					rules: [
+						{
+							type: 'quantity',
+							maxSize: '1'
+						}
+					],
+					isUsable: 'true',
+					weight: 20,
+
+					caption: function(node, caption_map) {
+						return (node.getMetadata('alert_info') || '');
+					},
+
+					edit: function(node, callback) {
+						var popup_html = $(self.getTemplate({
+								name: 'set-ringtone',
```

Comment:
Please rename the template so that it matches the module it corresponds to:
```suggestion
								name: 'setAlertEdit',
```